sdks/python/tox.ini

```diff
@@ -410,8 +410,9 @@ deps =
     tensorflow>=2.12rc1,<2.13
     # Help pip resolve conflict with typing-extensions for old version of TF https://github.com/apache/beam/issues/30852
     pydantic<2.7
-    protobuf==4.25.5
-extras = test,gcp,ml_test
+extras = test,gcp
```

Comment:
Tox honors the package deps with this order: deps -> extras -> install_requires. So if tensorflow is set several times, only the last one takes the effect. @jrmccluskey FYI.

Reply:
we remove `ml-test` here since these tests only rely on tensorflow.

Reply:
Gotcha, I see